.appveyor.yml

```diff
@@ -1,6 +1,10 @@
 environment:
   matrix:
     - nodejs_version: "10"
+    - nodejs_version: "11"
+    - nodejs_version: "12"
+    - nodejs_version: "13"
+    - nodejs_version: "14"
```

Comment:
I'm not sure what Node.js versions the Playwright team wants to officially support, so please modify this list as necessary.